README.md

```diff
@@ -64,6 +93,34 @@ considered forbidden by the license classifer. See
 
 for licenses considered forbidden.
 
+## Usages
+
+Report usage:
+
+```shell
+go-licenses csv <package>
+```
+
+Complying with license terms usage:
```

Comment:
```suggestion
Save licenses, copyright notices and source code (depending on license type):
```

I'd reword this since usage of this tool doesn't guarantee license compliance - e.g. edge cases like special license terms may result in ambiguous cases where the tool doesn't know what to do.